dev_tools/packaging/create_pypirc.sh

```diff
@@ -0,0 +1,9 @@
+echo """
```

Comment:
Can you add some comments to explain what this shell script is doing, for future reference?